src/troubleshooting/general/magento-2.4.0-checkout-error-selecting-local-payments.md

```diff
@@ -3,7 +3,9 @@ title: "Magento 2.4.0: Checkout error selecting local payments"
 labels: 2.4.0,Braintree,Magento Commerce,Magento Commerce Cloud,billing address,checkout,known issues,payment method
 ---
 
-This article talks about a solution for known issue in Magento during checkout where an error message appears when selecting a local payment method for some countries. This occurs for the countries Belgium, Italy, Netherlands, Poland, and Spain. The error message, " *There are currently no available payment methods. Please update your Billing Address.* " will appear, but the local payment methods will still appear and function correctly. A permanent fix will be available in Magento 2.4.1.
+This article talks about a solution for known issue in Magento during checkout where an error message appears when selecting a local payment method for some countries. This occurs for the countries Belgium, Italy, Netherlands, Poland, and Spain. 
```

Comment:
```suggestion
This article talks about a solution for known issue in Adobe Commerce during checkout where an error message appears when selecting a local payment method for some countries. This occurs for the countries: Belgium, Italy, Netherlands, Poland, and Spain. 
```

Reply:
Reject : The problem here is that "Adobe Commerce" is the licensed version. The free version is "Magento Open Source" - making the change above implies that it is only for the version. In this case Magento is generic.

Colon not appropriate here.

Reply:
Unfortunately, in the Support KB (you are welcome to verifying this with Sasha/Stephanie/Vicky), we do not mention Magento Open Source unless it is for a specific issue with Magento Open Source. There are several reasons why it isn't specifically mentioned in every article, including the implication that mentioning Magento Open Source in all articles implies that it is supported by Support engineers-- and Magento Open Source is not supported by Support engineers -- plus, the rebrand is to change the "Magento" generic term to "Adobe Commerce".

The colon is proper English grammar preceding a list. https://grammar.yourdictionary.com/punctuation/grammar-colon-usage.html (among other places) -- Reference Rule 1.